lib/config/executor/container_vm.go

```diff
@@ -143,6 +143,9 @@ type ExecutorConfig struct {
 
 	// version
 	Version *version.Build `vic:"0.1" scope:"read-only" key:"version"`
+
+	// migrated - this is used during in memory migration to assign whether an execConfig is viable for a commit phase
+	Migrated bool
```

Comment:
Here should be the one you want to change: https://github.com/vmware/vic/blob/master/lib/portlayer/exec/container.go#L136